examples/timeline/markers/customTimeMarkers.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Timeline | Markers example</title>
+
+  <style type="text/css">
+    body, html {
+      font-family: sans-serif;
+      font-size: 11pt;
+    }
+  </style>
+
+  <script src="../../../dist/vis-timeline-graph2d.min.js"></script>
+  <link href="../../../dist/vis-timeline-graph2d.min.css" rel="stylesheet" type="text/css" />
+
+</head>
+<body>
+
+<p>The Timeline has a function to add multiple custom time markers which can be dragged by the user.</p>
+<p>In this example, you can add new markers by double-clicking the timeline below and delete the markers by double-clicking it.</p>
+<p>
+  <input type="text" id="markerText" placeholder="marker text">
+</p>
+
+<div id="visualization"></div>
+
+<script type="text/javascript">
+  var container = document.getElementById('visualization');
+  var items = new vis.DataSet();
```

Comment:
It would be nice if some dummy-data would be displayed.
Something like this 
```js
var items = [
    {x: '2014-06-11', y: 10},
    {x: '2014-06-12', y: 25},
    {x: '2014-06-13', y: 30},
    {x: '2014-06-14', y: 10},
    {x: '2014-06-15', y: 15},
    {x: '2014-06-16', y: 30}
  ];
  var dataset = new vis.DataSet(items);
```

Reply:
I think too.
However, I'm not sure if it would be nice to use `vis.Graph2D` instead of `vis.Timeline` like [this](http://jsfiddle.net/qwapcbve/) since this examples are the timeline documents, not the graph2d documents.

Reply:
you are write, I copied the wrong example code. Feel free to add some data-points or leave it as it is.